map-unordered/CHANGELOG.md

```diff
@@ -0,0 +1,12 @@
+Change log
+==========
+
+`map-containers` uses [PVP Versioning][1].
```

Comment:
I think it should be `map-unordered` here. At this point I have no Idea how to maintain changelog for so many packages... So let's not do this but at least have proper names. We're waiting for _multiple public packages_ feature to be implemented in `cabal` and this will make our lives easier.